openstates/me/bills.py

```diff
@@ -194,21 +194,31 @@ def scrape_bill(self, bill):
         if srow:
             bill['subjects'] = [s.strip() for s in srow if s.strip()]
 
+        # Attempt to find link to bill text/documents.
         ver_link = page.xpath("//a[contains(@href, 'display_ps.asp')]")[0]
         ver_url = ver_link.get('href')
+
         try:
             ver_html = self.get(ver_url, retry_on_404=True).text
```

Comment:
Thanks, I'll keep that in mind if I run into these cases again.  Also, if you think it's better practice, feel free to edit it in before merging.